packages/material-ui-lab/src/TreeItem/TreeItem.js

```diff
@@ -189,6 +197,7 @@ const TreeItem = React.forwardRef(function TreeItem(props, ref) {
 
   const handleMouseDown = (event) => {
     if (event.shiftKey || event.ctrlKey || event.metaKey) {
+      // Prevent text selection
```

Comment:
Doesn't this also prevent focus in certain browsers or do those only focus if no meta keys are pressed?

Reply:
You're right but because we don't depend on the focusing of a tree item for anything since we call `focus` in the click handler it shouldn't matter.